sources/sns/shared.go

```diff
@@ -24,7 +24,11 @@ func tagsByResourceARN(ctx context.Context, cli tagLister, resourceARN string) (
 		return nil, err
 	}
 
-	return output.Tags, nil
+	if output != nil && output.Tags == nil {
```

Comment:
This logic is backwards right? shouldn't it be:

```suggestion
	if output != nil && output.Tags != nil {
```

Reply:
Thanks!

Reply:
https://github.com/overmindtech/aws-source/pull/421/commits/c4972c6a562315206e62594c0fa0a3ec126c212f